pxtblocks/plugins/duplicateOnDrag/dragStrategy.ts

```diff
@@ -5,6 +5,7 @@
  */
 
 import * as Blockly from "blockly";
+import { ConnectionPreviewer } from "../renderer";
```

Comment:
left behind from some refactor?

Reply:
Left behind from a more complex way of doing something I think. Removed here [ae4f56b](https://github.com/microsoft/pxt/pull/10512/commits/ae4f56b41fce1373e0a454953143a3cc70dfe607)